next.config.js

```diff
@@ -3,6 +3,6 @@ module.exports = {
     webpack5: true,
   },
   images: {
-    domains: ["raw.githubusercontent.com"],
+    domains: ["raw.githubusercontent.com","res.cloudinary.com"],
```

Comment:
Should we not remove github content URL, now that we have cloudinary?

Reply:
In Challenges, we are using GitHub content URLs. The challenge is not working correctly. I tried looking into it but the data model coming from the backend isn't correct. Hence apart from that, I changed in the remaining places i.e idle members, available tasks etc.